derive/src/to_client.rs

```diff
@@ -78,7 +78,7 @@ pub fn generate_client_module(
 				#(#client_methods)*
 			}
 
-			impl#generics From<RpcChannel> for Client#generics
+			impl #generics From<RpcChannel> for Client #generics
```

Comment:
What would happen if you did not make these whitespace changes?

Reply:
That is done by cargo fix.

See: https://doc.rust-lang.org/edition-guide/rust-2021/reserving-syntax.html